app/components/navbar/navbar_component.rb

```diff
@@ -10,7 +10,7 @@ def nav_link(link_text, link_path)
       class_name = "active" if first_uri_segment_matches_link?(link_path)
 
       content_tag(:li, class: class_name) do
-        link_to link_text, link_path
+        link_to_unless_current link_text, link_path
```

Comment:
Do we need to replicate this in the mobile nav bar?

Reply:
Ah didn't realise the mobile one wasn't using the `nav_link` method, will change it and make sure it looks ok